src/modules/dashboard/components/ColonyTokenManagementDialog/tokens/tokenList.mainnet.json

```diff
@@ -0,0 +1,485 @@
+[
+  {
+      "chainId":1,
```

Comment:
Do we still need `chainId` here? seeing as it's not in use and the other lists don't have it?